contrib/test/integration/main.yml

```diff
@@ -93,6 +93,14 @@
       include: "build/runc.yml"
       when: "build_runc | bool"
 
+    - name: set fs.may_detach_mounts = 1
+      sysctl:
+        name: fs.may_detach_mounts
+        state: present
+        value: 1
+        sysctl_set: yes
+        ignoreerrors: yes
```

Comment:
ignoreerrors boolean
Choices:
no ←
yes
Use this option to ignore errors about unknown keys.

Alas, this is not working for some reason

Reply:
Maybe it's because of `state: present` stance. Let me try removing it.

Reply:
retrying with `state: absent`.